2021/Robot2021/src/main/cpp/commands/IndexerDeploy.cpp

```diff
@@ -26,7 +27,7 @@ IndexerDeploy::IndexerDeploy(bool indexer) : m_indexer(indexer)
 // Called just before this Command runs the first time
 void IndexerDeploy::Initialize()
 {
-    std::printf("2135: IndexerDeploy - Init (%d)\n", m_indexer);
+    spdlog::info("2135: IndexerDeploy - Init {}", m_indexer);

```

Comment:
I think we can take out the 2135: because it already shows on the log, but everything else looks good!

Reply:
Thanks, I just fixed that!